bzt/engine.py

```diff
@@ -577,7 +578,11 @@ def load(self, config_files, callback=None):
                 configs = []
                 self.log.debug("Reading %s", config_file)
                 with open(config_file) as fds:
-                    configs.extend(yaml.load_all(fds))
```

Comment:
Probably, yes. I'm going to experiment with it while also testing that this approach works with Unicode.